src/tests/utests/iotc_utest_bsp_crypto_sha256.cc

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2018 Google LLC
```

Comment:
@verkri: So should we rather:
**A)** Merge all those tests into a single `src/bsp/tls/iotc_bsp_crypto_test.cc` file ?
**B)** Move them in `src/bsp/tls/` but still keep separate files, because a single one would be too large ?

Is there any recommendation about the threshold where the size becomes too large for a single file ?
And if we go with option **B)**, how should the files be named ? `src/bsp/tls/iotc_bsp_crypto_test_sha256.cc`, `src/bsp/tls/iotc_bsp_crypto_test_ecc.cc`, ... ?

Reply:
I think if the test file became too large, or e.g. many helper functions polluted the scope, it would make sense to separate it into multiple files.

Separating on type of encryption makes sense, so I like your **B)** approach and `src/bsp/tls/iotc_bsp_crypto_test_sha256.cc, src/bsp/tls/iotc_bsp_crypto_test_ecc.cc` separation.